examples/schema_site_template_contract_service_graph/main.tf

```diff
@@ -66,7 +66,7 @@ resource "mso_schema_site" "schema_site_1" {
   template_name = data.mso_schema_template_service_graph.sg1.template_name
 }
 
-resource "mso_site_template_contract_service_graph" "example" {
+resource "mso_schema_site_template_contract_service_graph" "example" {
```

Comment:
I think you changed it incorrect, where it should be `mso_schema_site_contract_service_graph`

Reply:
this applies to the entire name change